server/consumer.go

```diff
@@ -504,11 +504,7 @@ func (mset *stream) addConsumerWithAssignment(config *ConsumerConfig, oname stri
 
 	sampleFreq := 0
 	if config.SampleFrequency != _EMPTY_ {
-		var err error
-		sampleFreq, err = strconv.Atoi(strings.TrimSuffix(config.SampleFrequency, "%"))
-		if err != nil {
-			return nil, NewJSConsumerInvalidSamplingError(err)
-		}
+		sampleFreq, _ = strconv.Atoi(strings.TrimSuffix(config.SampleFrequency, "%"))
```

Comment:
Little comment above to explain that you know that it can't fail?

Reply:
fixed